utils/docker/images/install-dml.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 #
 # SPDX-License-Identifier: BSD-3-Clause
-# Copyright 2021, Intel Corporation
+# Copyright 2021-2022, Intel Corporation
```

Comment:
General comment: In the run-build script we have COMPILE_DML=1, but I am not sure if we have default execution COMPILE_DML=0.